docs/source/index.rst

```diff
@@ -177,3 +177,14 @@ experiment with PyTorch.
     This example implements the `Semi-Supervised Classification with Graph Convolutional Networks <https://arxiv.org/pdf/1609.02907.pdf>`__ paper on the CORA database.
 
     `GO TO EXAMPLE <https://github.com/pytorch/examples/blob/main/gcn>`__ :opticon:`link-external` 
+
```

Comment:
Doc build is failing, most likely culprit is changes to this file

Reply:
Thanks for the review. I rechecked that.
My local build is served without any issues with sphinx in venv.
i'll keep looking to see where could the issue be